### ISSUE TITLE
Crash on OSX 10.5.8

### ISSUE THREAD

**Issue by GoogleCodeExporter** [kwin/macdependency#3]
```
What steps will reproduce the problem?
1. Just open a library with File -> Open...

What is the expected output? What do you see instead?

I presume I should see the main app window. Instead, I get a message from the 
Operating System (OSX 10.5.8) saying that the app has crashed


What version of the product are you using? On what operating system?

version 1.0.0, on Mac OSX 10.5.8

Please provide any additional information below.

It looks like it is a segmentation fault. Attached you may find the crash 
report text from the system.

```

Original issue reported on code.google.com by `ludola...@gmail.com` on 18 Jun 2012 at 2:20

Attachments:
- [crash_report.txt](https://storage.googleapis.com/google-code-attachments/macdependency/issue-3/comment-0/crash_report.txt)

**Comment by kwin** [comment]
The first crash is probably be resolved with https://github.com/kwin/macdependency/commit/972fcb30943d016dd3a665e062bb56d75b431503 the 2nd one is resolved with the same commit as well, as boost process is no longer being used for demangling the function names.

**Comment by barracuda156** [comment]
@kwin A switch to 10.7 target and C++11 (which is not supported by Xcode of 10.5) cannot possibly fix the app on 10.5.8, it will just break the build.

(As such, C++11 and later standards can be supported on 10.5 without issues, but not via Xcode build system.)
